app/javascript/mastodon/features/about/index.jsx

```diff
@@ -71,15 +71,13 @@ class Section extends PureComponent {
     const { collapsed } = this.state;
 
     return (
-      <div className={classNames('about__section', { active: !collapsed })}>
-        <div className='about__section__title' role='button' tabIndex={0} onClick={this.handleClick}>
+      <details open={!collapsed} onOpen={this.handleClick} className={classNames('about__section', { active: !collapsed })}>
```

Comment:
The correct event is `toggle`, not `open`, so `onOpen` results in the callback not being called.

`details` seems tricky to use as a managed component: https://github.com/facebook/react/issues/15486#issuecomment-873516817

Reply:
my bad, thanks for the review and the link, I should have watched the terminal logs.

I've made the change, I used the event state to make sure the react state is synchronised https://github.com/facebook/react/issues/15486#issuecomment-1537086764 

If you want we could also just use a `button` instead of a `details` element with an `aria-expanded`